Vote.cs

```diff
@@ -19,6 +19,10 @@ public class Vote : IVote
 
         public VoteStatus Status { get; set; } = VoteStatus.Ready;
 
+        public Coroutine thisCoroutine1 { get; set; }
+
+        public Coroutine thisCoroutine2 { get; set; }
+
```

Comment:
Perhaps these should be private and named properly?
```suggestion
        private Coroutine _startCoroutine;

        private Coroutine _cooldownCoroutine;

```